packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -89,12 +90,15 @@ class ProgressivePaginator extends React.Component {
       }
 
       this.props.onPageChange(index);
-
       let message;
       if (totalCount && itemCountPerPage) {
-        message = intl.formatMessage({ id: 'Terra.paginator.pageSelectedWithCount' }, { pageNumber: index, pageNumberTotal: calculatePages(totalCount, itemCountPerPage) });
+        if (pageLabel) {
```

Comment:
```suggestion
        if (pageLabel && !hidePageCount) {
```
can we add `hidePageCount` to as well to prevent conflict between scenario with default page count and custom pagecount